mail-filter/mimedefang/files/mimedefang.conf

```diff
@@ -0,0 +1,131 @@
+# Additional options for MIMEDefang.
+# read man pages mimedefang(8) for additional informations.
+
+# Locale should be set to "C" for generating valid date headers
+LC_ALL=C
+export LC_ALL
+
+# The settings which follow are defaults.  You can override them by
+# placing assignments in /etc/mail/mimedefang.conf
+
+# The socket used by mimedefang to communicate with sendmail
+# SOCKET=$SPOOLDIR/mimedefang.sock
+
+# Run the multiplexor and filters as this user, not root.  RECOMMENDED
+MX_USER=defang
```

Comment:
To be honest, I find it really silly to invent new variables for all command line switches. That just makes Gentoo admins have to learn second set of ways of doing everything that doesn't really benefit them long term. I'd just go for some variable to pass options, and a lot of `FOO+="-U foo"` examples in the config file, e.g.:

```
# Run the multiplexor and filters as this user, not root.  RECOMMENDED
opts+='-U defang'

# Syslog facility
# opts+='-S mail'

Reply:
Please see new commit. I couldn't use arrays without running into bashism. It would've been better (less error-prone) to use +=('') and then "$foo[@]".

Reply:
Yes, we had this weird idea to switch OpenRC to pseudo-POSIX shell for no particular reason. Now we have to live with the consequences.